visual-testing-app/src/components/info-detail-page/info-detail-page.visualroute.tsx

```diff
@@ -0,0 +1,59 @@
+import Text from '@commercetools-uikit/text';
+import { InfoDetailPage } from '@commercetools-frontend/application-components';
+import { Suite, Spec } from '../../test-utils';
+
+export const routePath = '/info-detail-page';
+
+type ContainerProps = Partial<Parameters<typeof InfoDetailPage>[0]>;
+
+const DetailPageContainer = (props: ContainerProps) => (
+  <div style={{ position: 'relative', height: '750px' }}>
+    <InfoDetailPage
+      title="Lorem ipsum"
+      subtitle="Lorem ipsum dolor sit amet, consectetur adipiscing elit."
+      {...props}
+    >
+      {props.children}
+    </InfoDetailPage>
+  </div>
+);
+DetailPageContainer.displayName = 'DetailPageContainer';
+
+export const Component = () => (
+  <Suite>
+    <Spec label="InfoDetailPage" size="xl">
+      <DetailPageContainer onPreviousPathClick={() => alert('Go back clicked')}>
+        <Text.Body>
+          {`Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur nec turpis in risus elementum fringilla. Vestibulum nec vulputate metus, fringilla luctus nisl. Vestibulum mattis ultricies augue sagittis vestibulum. Nulla facilisi. Quisque tempor pulvinar efficitur. Praesent interdum ultrices leo. Vivamus non ex maximus justo egestas suscipit eget sed purus. Aliquam ut venenatis nulla. Fusce ac ligula viverra, blandit augue eget, congue turpis. Curabitur a sagittis leo. Nunc sed quam dictum, placerat nunc quis, luctus erat.`}
```

Comment:
I don't think we need tow rap these in `{``}`. There are some places in this PR we do so and others we don't.